infra/app/env-template/main.tf

```
@@ -303,6 +303,7 @@ module "staff_secret" {
 }
 
 resource "aws_ssm_parameter" "staff_jwt_secret" {
+  # checkov:skip=CKV_AWS_337:Need to assign the access permissions for the KMS key, which would be too much lift for the time remaining
```

Let's change all of these notes to something more friendly for users that might approach this fresh.

```suggestion
  # checkov:skip=CKV_AWS_337:Skip creating separate IAM roles for KMS keys
```